src/styles/index.scss

```diff
@@ -1,3 +1,65 @@
+@import '../utils/variables';
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  box-sizing: border-box;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), 'box-sizing: border-box;' should be included globally for all elements using the '*' selector to avoid repeating it for individual elements.